test/create3/Create3Factory.t.sol

```diff
@@ -105,11 +107,12 @@ contract Create3FactoryTest is DeploymentHelper, TestSetup {
     function testFuzz_DeployOrderIndependence(uint256 pkNum) public {
         // Setup
         VmSafe.Wallet memory wallet = vm.createWallet(uint256(keccak256(abi.encodePacked(uint256(pkNum)))));
+        bytes memory constructorArgsBytecode = abi.encode(wallet.addr);
 
         // First deployment set
         uint256 snapShot = vm.snapshot();
-        deployChild(address(proxy), wallet, strippedBytecodeChildA);
-        deployChild(address(proxy), wallet, strippedBytecodeChildB);
+        deployChild(address(proxy), wallet, strippedBytecodeChildA, constructorArgsBytecode);
+        deployChild(address(proxy), wallet, strippedBytecodeChildB, "");
```

Comment:
Shouldn't this be "0x"?

Reply:
According to [this](https://ethereum.stackexchange.com/questions/6618/in-solidity-how-can-you-set-a-bytes-variable-to-empty), no.